### PR TITLE
[WIPTEST] Test delete all actions from compliance policy

### PR DESCRIPTION
**Pull request by quarckster**
Purpose or Intent
=================

__New test__ for Control which tests deleting all actions from a compliance policy. We should not allow a compliance policy to be saved if there are no actions on the compliance event (since you can't add/remove events from compliance policies). https://bugzilla.redhat.com/show_bug.cgi?id=1395965

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 2955431bd554e4ad65ec54afec4ea2689097e69e:

Lint OK! :cake: :punch: :cookie: .


**No commit flaws detected**.
<mergeable/>

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit e8adff1c48c90d62dcbf81159c3e04a270cad7ea:

Lint OK! :cake: :punch: :cookie: .


**No commit flaws detected**.
<mergeable/>

*CFME QE Bot*

**Comment by quarckster** [comment]
Checks have failed because the bug hasn't been fixed in downstream yet.

File: cfme/tests/control/test_bugs.py [hunk 1]
```diff
@@ -172,3 +173,25 @@ def _delete_action():
...
+
+    @request.addfinalizer
+    def _delete_policy():
+        if policy.exists:
+            policy.delete()
+
+    policy.create()
+    # Until bug is not fixed we will match any character
+    with error.expected(".*"):
+        policy.assign_actions_to_event("VM Compliance Check", [])
```
Comment (mfalesni) [inline cfme/tests/control/test_bugs.py]: That fails matching a flash message, right?

What about?

```python
from cfme.exceptions import FlashMessageException

with pytest.raises(FlashMessageException):
    policy.assign_actions_to_event("VM Compliance Check", [])
```

@RonnyPfannschmidt @psav ?

Comment (quarckster) [inline cfme/tests/control/test_bugs.py]: Definitely it looks nicer.

File: cfme/tests/control/test_bugs.py [hunk 2]
```diff
@@ -174,3 +175,25 @@ def _delete_action():
...
+
+@pytest.mark.meta(blockers=[1395965], automates=[1395965])
+def test_delete_all_actions_from_compliance_policy(request):
+    """We should not allow a compliance policy to be saved
+    if there are no actions on the compliance event.
+    Steps:
+        * Create a compliance policy
+        * Remove all actions
+    The policy shouldn't be saved.
+    """
```
Comment (mfalesni) [inline cfme/tests/control/test_bugs.py]:
```python
   """We should not allow a compliance policy to be saved
    if there are no actions on the compliance event.

    Steps:
        * Create a compliance policy
        * Remove all actions

    Result:
        The policy shouldn't be saved.
    """
```

**Comment by dajoRH** [comment]
Lint report for commit 0d944bbce99e2338c2d8b4ff137b1c1e97eba587:

`cfme/tests/control/test_bugs.py`:
- :heavy_exclamation_mark: Line 185:1: **W293** *blank line contains whitespace*

Please, rectify these issues :smirk: .

* 0d944bbce99e2338c2d8b4ff137b1c1e97eba587
  * Merge commit detected!!! :bangbang:

**The PR might not be acceptable in this state** :rage4:.
There is a merge commit in the branch which we do not like in PRs so please, get rid of it.

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit d19a56538e375a614b3270d3b1897920f8b98a6a:

`cfme/tests/control/test_bugs.py`:
- :heavy_exclamation_mark: Line 184:1: **W293** *blank line contains whitespace*

Please, rectify these issues :smirk: .

* d19a56538e375a614b3270d3b1897920f8b98a6a
  * Merge commit detected!!! :bangbang:

**The PR might not be acceptable in this state** :rage4:.
There is a merge commit in the branch which we do not like in PRs so please, get rid of it.

*CFME QE Bot*

**Pull request closed without merging**